tests/run_tests.py

```diff
@@ -9,13 +9,17 @@
 import datetime
 from collections import OrderedDict
 import string
+import numpy as np
+import xarray as xr
+import numpy.testing as npt 
 
 import pytest
 import pandas as pd
 
 from tonic.models.vic.vic import VIC, VICRuntimeError  # , read_vic_ascii
 from tonic.io import read_config, read_configobj
 from tonic.testing import check_completed, check_for_nans, VICTestError
+from test_image_driver import assert_nan_equal 
```

Comment:
Hmm I think this way of calling functions from another file only works when you run the `run_tests.py` script from the directory where the scripts are. @jhamman do we want to put some functions in a separate file, and then call it in some more general way (how?)? Or do we want to just put all the functions in `run_tests.py`?